api/api/utils/exceptions.py

```diff
@@ -1,4 +1,4 @@
-from rest_framework.serializers import ValidationError
+from rest_framework.exceptions import ValidationError
```

Comment:
Please correct me if I'm mistaken but I thought the practice was to raise `ValidationError` from `rest_framework.serializers` as per [these docs](https://www.django-rest-framework.org/api-guide/serializers/).

Reply:
`serializers.ValidationError` is an alias for the `exceptions.ValidationError`. The only good reason to use `serializers.ValidationError`, as in the docs, is because the convention in _serializer definition modules_ is to `from rest_framework import serializers`, and so it's convenient and consistent to address it from the alias in that module. There's no reason not to use the exceptions module, though, and I think it makes particular sense in a file that otherwise has nothing to do with serialization (it's named `exceptions.py`).

At least, that's my interpretation of why the docs are written that way, and why I would choose to write code one way or the other, but I also don't think there's a reason for/against always using one or the other, and there's nothing in the docs you linked that explicitly implies you should use one or the other, aside from the examples using the alias (I suspect for the reason above).